modular_skyrat/modules/bodyparts/code/vox_bodyparts.dm

```diff
@@ -9,6 +9,9 @@
 	bodyshape = parent_type::bodyshape | BODYSHAPE_SNOUTED | BODYSHAPE_CUSTOM
 	limb_id = SPECIES_VOX
 
+/obj/item/bodypart/chest/mutant/vox/get_butt_sprite()
+	return icon('modular_skyrat/master_files/icons/mob/butts.dmi', BUTT_SPRITE_VOX)
```

Comment:
can you make this a define on the chest instead pls

Reply:
Like this?

Reply:
no no, the actual icon, like, how it returns icon('whatever'), make the icon a var on the chest, e.g. butt_sprite = 'icon', so we can just change the var, maybe put the icon state there too

Reply:
Would here be better, so it's consistent with TG? https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/code/__DEFINES/mobs.dm#L1013 I can

Reply:
nvm i read it wrong lmao i need sleep, itll be fine